### PR TITLE
Update Set-AzActivityLogAlert.md

### PR DESCRIPTION
**Pull request by thomast1906**
cmdlet is New-AzActivityLogAlertCondition not Get-AzActivityLogAlertCondition 

Have updated URL and it works correctly.
- Previous loaded 404

## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

File: src/Monitor/Monitor/help/Set-AzActivityLogAlert.md [hunk 1]
```diff
@@ -459,4 +459,4 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [New-AzActionGroup](./New-AzActionGroup.md)
 
-[New-AzActivityLogAlertCondition](./Get-AzActivityLogAlertCondition.md)
+[New-AzActivityLogAlertCondition](./New-AzActivityLogAlertCondition.md)
```
Comment (markcowl) [inline src/Monitor/Monitor/help/Set-AzActivityLogAlert.md]: Please add an entry to the chagelog for this change, per the PR checklist

Comment (thomast1906) [reply src/Monitor/Monitor/help/Set-AzActivityLogAlert.md]: Have now updated change log, let me know if the format is Ok in changelog.md

File: src/Monitor/Monitor/ChangeLog.md [hunk 2]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.2.2
```
Comment (markcowl) [inline src/Monitor/Monitor/ChangeLog.md]: This just goes under the 'Upcoming release' heading, and our versioning tools will determien the new version number, per the comments above in the file.

**Comment by markcowl** [comment]
@thomast1906 Just need to update the changelog change slightly.

**Pull request merged**